source/includes/_resource.collections.md

```diff
@@ -547,7 +557,7 @@ GET /collections/:id/items
 ```
 
 ```ruby
-Vhx::CollectionItem.list()
+Vhx::Item.all()
```

Comment:
interesting. hmmmmmmmmmm

let's stick with collection.items and video.files for now? only expose those larger objects later if need be?